kitsune/users/templates/users/profile.html

```diff
@@ -64,10 +64,15 @@ <h2 class="location">
         {% endif %}
       {% endif %}
 
-      {% if num_answers or num_questions or num_solutions or num_documents %}
+      {% if num_answers or num_questions or num_solutions or num_documents or profile.involved_from %}
         <section class="contributions">
           <h2>Contributions</h2>
           <ul>
+            {% if profile.involved_from %}
+              <li>
+                {{ _('Started on') }} {{ profile.involved_from.strftime('%b %Y') }}
```

Comment:
This works for English, but doesn't work for some other languages that may want to put the date somewhere besides the end of the sentence. This should be something like `{{ _('Involved with Mozilla since {date}')|f(date=...) }}`

Also, using strftime directly like this doesn't localize well. I'm not sure what the right thing to do about that is though. Can you look into that?